package.json

```diff
@@ -13,15 +13,16 @@
     "build": "ng build --build-optimizer --delete-output-path --optimization --progress --vendor-chunk --aot",
     "watch": "ng build --watch --configuration development",
     "sonar-scanner": "npm run test && sonar-scanner",
-    "security-checks": "git secrets --scan && git log -p | scanrepo",
```

Comment:
I'd say we need to keep scanrepo until we are officially told not to use it anymore.

Reply:
I agree